### ISSUE TITLE
gftl-shared with CCE?

### ISSUE THREAD

**Issue by AlexanderRichert-NOAA** [Goddard-Fortran-Ecosystem/gFTL-shared#60]
I'm trying to compile gFTL-shared with Cray compiler on one of the NOAA systems, and I'm getting a lot of errors. Is it intended to support the Cray compiler, or if not, is there an plan to? I'm happy to provide more detail about error messages etc.

**Comment by tclune** [comment]
Yes - to both.  At one point the Cray compiler could support pFUnit but the support was not maintained as I don't have direct access to a Cray compiler.   But just  last month, a user reached out to me with saying that he was working with Cray on a port:  "The Cray chap, Ian Cockshot, should hopefully be creating a branch off of the pfunit repo at some point and adding in the changes. As mentioned it is a bit of a hack so you will want to check things."

We've already identified a portability bug in the MPI launch and I'm planning to merge that change very soon.   But I have not yet seen any of the changes mentioned above.    Even if I regard some of the code changes to be unacceptable for the main release, I'm happy to maintain a side branch for Cray so long as there are users that can actively check new releases and report back.
